domain-rrd/rrd-shared/src/main/java/gov/va/vro/model/rrd/bgs/BgsApiClientRequest.java

```diff
@@ -22,4 +22,9 @@ public class BgsApiClientRequest {
 
   /** Multiple claim notes can be submitted */
   public List<String> claimNotes = new ArrayList<>();
+
+  public boolean isConstraintSatisfied() {
+    if (veteranNote == null) return vbmsClaimId != null && !claimNotes.isEmpty();
+    else return veteranParticipantId != null && claimNotes.isEmpty();
+  }
```

Comment:
Only used for testing but I thought it'd be good to include to encode this constraint in this class.